### ISSUE TITLE
Can't open store in Window Phone IE

### ISSUE THREAD

**Issue by YodasWs** [YodasWs/incremental-game#8]
Tap the button to open the store in IE on Windows Phone 8.1, and the store does not appear to open.

WinPhone IE does have a history of distorting the vertical scroll and top positions (i.e., the supposed length from the top of the page). Probably should be using a fixed position instead of an absolute anyhow.

**Comment by YodasWs** [comment]
And with the update in 07db997, clicking the store button does attempt to open the store as evidenced by the `#modal-bg` showing, but tapping that does not remove it. This would indicate that the store is not being shown even off screen. I.e., the bug is within `game.openShop` after `game.showModalBG(t)` and before `Z('#shop').show()`.

Chrome does complain about an undefined `item.level` at the point we're supposed to redefine it. And looking at that point in `game.updateShopItem` I see the first two lines need to be switched. Hopefully that is the only faulty point for IE.

**Comment by YodasWs** [comment]
And no, apparently the switch in 6bed2a5 was not enough to fix this IE problem.

**Comment by YodasWs** [comment]
While looking to add images of the store items, I suddenly remembered that many browsers don't like the last item in a list/object to have a comma:

``` javascript
{
    name:'Carrots',
    baseCost:5,
    bonus:1,
}
```

If I can get this app to work in IE8, then it should work in Safari and IE11.

**Comment by YodasWs** [comment]
Wanting to mimic/utilize PhoneGap's remote server error logging, I updated the app in 8affe77 and the results are beautiful:

```
Object doesn't support property or method 'parseInt'
```

**Comment by YodasWs** [comment]
That was it. Problem solved in f335360.
